packages/plugins/integration-zapier-ui/src/lib/components/zapier-actions/zapier-actions.component.ts

```
@@ -0,0 +1,137 @@
+		private readonly _toastrService: ToastrService,
+		private readonly _store: Store,
+		private readonly _activatedRoute: ActivatedRoute,
+		public readonly translateService: TranslateService
+	) {
+		super(translateService);
+	}
+
+	ngOnInit(): void {
+		// Subscribe to route parameters to get integration ID
+		this._activatedRoute
+			.parent!.params.pipe(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)